homeassistant/components/alexa/entities.py

```diff
@@ -138,7 +138,8 @@ def friendly_name(self):
 
     def description(self):
         """Return the Alexa API description."""
-        return self.entity_conf.get(CONF_DESCRIPTION, self.entity.entity_id)
+        description = self.entity_conf.get(CONF_DESCRIPTION) or self.entity_id
+        return f"{description} via Home Assistant"
```

Comment:
If a description is set, we shouldn't add text to it.

Reply:
It is required by Alexa to show how it is connected, just like it is required to include "scene" for scenes.

Reply:
ah okay.

Can we also filter out characters that are not allowed ?